ydb/core/tx/schemeshard/schemeshard__table_stats.cpp

```diff
@@ -205,14 +231,17 @@ bool TTxStoreTableStats::PersistSingleStats(const TPathId& pathId,
     }
 
     TShardIdx shardIdx = Self->TabletIdToShardIdx[datashardId];
+    const auto& shardInfo = Self->ShardInfos[shardIdx];
```

Comment:
@ijon What if the datashard has migrated since the TEvPeriodicTableStats message was sent? Will we **always** get the correct shardInfo here?